packages/database/src/core/view/QueryParams.ts

```diff
@@ -53,8 +54,10 @@ const enum REST_QUERY_CONSTANTS {
   PRIORITY_INDEX = '$priority',
   VALUE_INDEX = '$value',
   KEY_INDEX = '$key',
+  START_AFTER = 'startAfter',
   START_AT = 'startAt',
   END_AT = 'endAt',
+  END_BEFORE = 'endBefore',
   LIMIT_TO_FIRST = 'limitToFirst',
   LIMIT_TO_LAST = 'limitToLast'
 }
```

Comment:
It might be worth adding a field-level comment that `startAfterSet_` is only true if `startAtSet_` is true. It has implications for synctree logic, per: https://github.com/firebase/firebase-js-sdk/blob/master/packages/database/src/core/view/QueryParams.ts#L181-L183.

Reply:
Done.